go/interpreter/lfvm/gas_test.go

```diff
@@ -59,3 +59,17 @@ func TestGas_CallGasCalculation(t *testing.T) {
 		})
 	}
 }
+
+func TestGas_CheckGasCostsForAllExecutableOpcodes(t *testing.T) {
+	for i := 0; i < int(NUM_OPCODES); i++ {
+
+		if static_gas_prices[i] == UNKNOWN_GAS_PRICE &&
+			op_2_op[i] != INVALID {
```

Comment:
As mentioned in other PRs, `op_2_op` is an internal utility of the converter and not supposed to be used as an universal mapping between EVM and LFVM op codes.